tools/coqmod/lexer.mll

```diff
@@ -68,7 +68,8 @@ let coq_ident_char = ['A'-'Z' 'a'-'z' '_' '\'' '0'-'9' '\128'-'\255']
 let coq_ident = coq_ident_start_char coq_ident_char*
 let coq_field = '.' coq_ident
 let coq_qualid = coq_ident coq_field*
-let coq_qid_quot = '"' coq_qualid '"'
+
+let coq_mod = ['A'-'Z' 'a'-'z' '.' ':' '-']+
```

Comment:
This is missing '_' and numbers of `ltac2_plugin` can be parsed, anyways I went for the hack.